exploration/number-selection.md

```diff
@@ -0,0 +1,179 @@
+# Selection on Numerical Values
+
+Status: **Accepted**
+
+<details>
+	<summary>Metadata</summary>
+	<dl>
+		<dt>Contributors</dt>
+		<dd>@eemeli</dd>
+		<dt>First proposed</dt>
+		<dd>2023-09-06</dd>
+		<dt>Pull Request</dt>
+		<dd><a href="https://github.com/unicode-org/message-format-wg/pull/471">#471</a></dd>
+	</dl>
+</details>
+
+## Objective
+
+Define how selection on numbers happens.
+
+## Background
+
+As discussed on numerous past occasions and explicitly identified in
+<a href="https://github.com/unicode-org/message-format-wg/pull/457">#457</a>,
+it would be good for both plural and non-plural match selection to be supported in MF2.
```

Comment:
This is slightly misleading--it implies a binary plural/non-plural set of selection, whereas we go on to point out that there are a variety of number selection needs. Perhaps:

```suggestion
As discussed by the working group and explicitly identified in
<a href="https://github.com/unicode-org/message-format-wg/pull/457">discussion of #457</a>,
there is a need to support multiple types of selection on numeric values in MF2.
```